enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/InteractiveDashboard.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+import { PublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
+
+export const StyledPublicComponentWrapper = styled(PublicComponentWrapper)`
```

Comment:
It seems like we're using this in a couple of places, do we have a semantic name for it?

Is it possible to solve this by not extending `PublicComponentWrapper`?
I thought that component was only supposed to be added with `withPublicComponentWrapper` when exporting the sdk components. If we need to adjust its style I'm assuming there's something wrong with it